frontend/helpers/commerce-helpers.ts

```diff
@@ -6,3 +8,27 @@ export function computeDiscountPercentage(
    percentage = Math.round(percentage);
    return percentage;
 }
+
+export type ShopifyPrice = Pick<MoneyV2, 'amount' | 'currencyCode'>;
+
+export function formatShopifyPrice(money: ShopifyPrice) {
+   let textAmount: string;
+   if (typeof money.amount === 'number') {
+      textAmount = money.amount.toFixed(2);
+   } else {
+      textAmount = money.amount;
+   }
+   const formattedCurrency = formatCurrency(money.currencyCode);
+   return `${formattedCurrency}${textAmount}`;
```

Comment:
I'm curious why we do this with string concatenation. It makes sense for USD, but not for other currencies. Browsers natively support formatting currency (and putting the currency symbol in the correct spot), so I'd expect to use that, or maybe shopify has something similar in their SDK?

Reply:
Good suggestion! I've refactored it to use the browser api 👍